src/main/resources/BridgeUserDataDownloadService.conf

```diff
@@ -24,6 +24,7 @@ dev.sqs.queue.url = https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-UDD-
 uat.sqs.queue.url = https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-UDD-Request-uat
 prod.sqs.queue.url = https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-UDD-Request-prod
 
+local.synapse.map.table = local-exporter-SynapseTables
```

Comment:
I think we shouldn't include this config line, since everyone will need to have their own Synapse projects, and hence, their own Synapse DDB configs.